sdks/python/apache_beam/runners/interactive/caching/streaming_cache.py

```diff
@@ -304,6 +304,18 @@ def read(self, *labels):
       return iter([]), -1
     return StreamingCache.Reader([header], [reader]).read(), 1
 
+  @staticmethod
+  def sentinel_label():
```

Comment:
Rather than introduce a sentinel label, how about returning a dict from expand iff output_tags was manually specified (or, alternatively, something other than `{None}`)?

Reply:
Yeah that can work, I like that because it keeps the same semantics. I'll go with the {None} alternative because the output_tags are always manually specified in the from_runner_api_parameter method.